src/lib_c/aarch64-linux-gnu/c/intrinsics.cr

```diff
@@ -0,0 +1,20 @@
+lib Intrinsics
```

Comment:
Intrinsics aren't LibC bindings (i.e. calls to external symbols) but LLVM intrinsics (inlined calls); they won't be generated automatically; they aren't system dependent (but arch dependent).

I.e.: they have nothing to do under `LibC`, please put them back as `src/intrinsics.cr` or maybe `src/crystal/intrinsics.cr`, along with the flag macro to select either the u32 or u64 calls.